Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+#!groovy
+
+node {
+  stage 'Checkout'
+  checkout scm
+
+  stage 'Build Grafana' {
```

Comment:
`stage 'whatever'` shouldn't be a closure... its basically just its own sep function, used as a pipeline delimiter